Sources_v3/StreamChatUI/ChatChannel/ChatFileAttachmentListView+ItemView.swift

```diff
@@ -0,0 +1,179 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+extension ChatFileAttachmentListView {
+    open class ItemView: View, UIConfigProvider {
+        public var attachment: _ChatMessageAttachment<ExtraData>? {
+            didSet { updateContentIfNeeded() }
+        }
+
+        public var tapHandler: () -> Void = {}
+
+        public var actionIconSize = CGSize(width: 20, height: 20)
```

Comment:
It weird that we use constraints for more flexible layout, but in same time use hardcoded values for image size.
Why not rely on intrinsic size of image?

This will make it super easy for our user to change icon size by just providing different size image

Reply:
Good catch! 👍 Fixed in e51a602f9ea9e46b6e63e3f59fd090f4d3c54036